src/Strategies/DNSRecord.php

```diff
@@ -16,6 +18,23 @@ class DNSRecord implements StrategyInterface
      */
     public function verify(string $url, DomainVerifiableInterface $domainVerifiable)
     {
+        $dns = new Dns($url);
```

Comment:
I think we should add a new method `getTxtRecords`. It will help you easier to write unittest.